docs/sources/whatsnew/whats-new-in-v8-1.md

```diff
@@ -12,47 +12,60 @@ list = false
 
 > **Note:** This topic will be updated frequently between now and the final release.
 
-This topic includes the release notes for Grafana v8.1. For all details, read the full [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+Grafana 8.1 builds upon our promise of composable, open observability platform with new panels and extends functionality launched in Grafana 8.0. We’ve got new panels, including the Geomap panel, the Annotations panel, and some great updates to the Time Series panel. For enterprise customers we’ve got additions to fine grained access control, updates to the reporting scheduler and more. We’ve also got some new transformations, updates to data sources and more. Details below.
+
+We’ve summarized what’s new in the release here, but you might also be interested in the announcement blog post as well. If you’d like all the details you can checkout the complete [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```

Comment:
```suggestion
In addition to what we have summarized here, you might also be interested in the announcement blog post. For all the details, check out the complete [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```

Reply:
I am really happy to see this introduction! It sets up the readers expectations nicely

Reply:
It was all your idea - thank you!